classes/class-object-sync-sf-salesforce.php

```diff
@@ -817,6 +817,16 @@ public function get_authorization_code() {
 			),
 			$this->login_url . $this->authorize_path
 		);
+
+		if ( get_option( $this->option_prefix . 'pkce_support', false ) ) {
+			$challenge = $this->get_challenge();
+
+			$url = add_query_arg( [
+				'code_challenge' => $challenge['code'] || '',
```

Comment:
I think is javascript like fallback logic, it'll throw a warning if the key is unset so you should use null-coalesce, or check if the key `isset`.

Reply:
🤦